framework/src/ble_manager/ble_manager_state.c

```diff
@@ -50,7 +52,7 @@ ble_result_e blemgr_handle_request(blemgr_msg_s *msg)
 {
 	trble_result_e ret = TRBLE_FAIL;
 
-	BLE_LOG_INFO("[BLEMGR] T%d --> _handle_request[%d]\n", getpid(), msg->event);
+	BLE_LOG_DEBUG("[BLEMGR] T%d --> _handle_request[%d]\n", getpid(), msg->event);
```

Comment:
To use `getpid()`, please include `unistd.h`.

Reply:
I will add it